src/domains/idpay-app/14_eventhub.tf

```diff
@@ -0,0 +1,45 @@
+locals {
```

Comment:
I think we can remove this part of "locals" and move `resource "azurerm_eventhub_namespace_authorization_rule" "evh_namespace_access_key_00"` and `
resource "azurerm_key_vault_secret" "event_hub_root_key_idpay_00"` to 13_config_kafka_connectors.tf which is strictly related to root access key